src/Gui/InventorAll.h

```diff
@@ -31,20 +30,27 @@
 #include <GL/gl.h>
 #endif
 #include <Inventor/C/basic.h>
+#include "Inventor/MarkerBitmaps.h"

```

Comment:
InventorAll.h is part of the PreCompiled.h and there only 3rd party headers should be used because they normally never are very rarely change. Adding application specific headers there destroy the advantage of PCH because whenever touching one of the application headers all PCH must be rebuilt.

Reply:
Oops, this change should no have happened.